rubygems-basics.md

```diff
@@ -200,15 +200,6 @@ You can view the documentation for your installed gems with `ri`:
     subclass of MultiRBTree.
     -------------------------------------------
 
-You can view the documentation for your installed gems in your browser with
```

Comment:
Would it be welcomed to replace this with `gem install yard` and `yard server -g`?

Reply:
I am not sure that goes well with intention of the page "Use of common RubyGems commands".